kinto_wizard/__main__.py

```diff
@@ -88,13 +89,17 @@ def main():
         logger.info("Load YAML file {!r}".format(args.filepath))
         with open(args.filepath, 'r') as f:
             config = yaml.safe_load(f)
-            event_loop.run_until_complete(
-                initialize_server(
-                    async_client,
-                    config,
-                    bucket=args.bucket,
-                    collection=args.collection,
-                    force=args.force,
-                    delete_missing_records=args.delete_records
+            try:
+                event_loop.run_until_complete(
+                    initialize_server(
+                        async_client,
+                        config,
+                        bucket=args.bucket,
+                        collection=args.collection,
+                        force=args.force,
+                        delete_missing_records=args.delete_records
+                    )
                 )
-            )
+            except KintoBatchException:
```

Comment:
Shouldn't we log anything here?

Reply:
I believe kinto-http.py already does.

Reply:
Don't uncaught exceptions automatically cause a nonzero exit status? Why even bother catching it? If there is a reason, should we put a similar catch around `load` ?

Reply:
That's actually a good question, I am going to remove it and see if tests still pass

Reply:
If I do so then I don't have any return code.

Reply:
Do you mean that you don't know what return code the wrapper script will return? I just did a test and this produces a return code of 1:

```diff
diff --git a/kinto_wizard/__main__.py b/kinto_wizard/__main__.py
index 271c55d..164720e 100644
--- a/kinto_wizard/__main__.py
+++ b/kinto_wizard/__main__.py
@@ -103,3 +103,6 @@ def main():
             except KintoBatchException:
                 return 1
     return 0
+
+def ethan_main():
+    1/0
diff --git a/setup.py b/setup.py
index 6c53358..7fb085a 100644
--- a/setup.py
+++ b/setup.py
@@ -30,7 +30,8 @@ TEST_REQUIREMENTS = [
 
 ENTRY_POINTS = {
     'console_scripts': [
-        'kinto-wizard = kinto_wizard.__main__:main'
+        'kinto-wizard = kinto_wizard.__main__:main',
+        'kinto-ethan-is-wizard = kinto_wizard.__main__:ethan_main',
     ]
 }
 
```